packages/app-mobile/components/ButtonWithTooltip.tsx

```diff
@@ -0,0 +1,197 @@
+//
```

Comment:
Just name it "CustomButton" or something generic like this. That way we can add more functionalities to it without having to create separate classes like ButtonWithRoundBorderAndTooltip, ButtonWithIconAndNoTooltip, etc. More generic is better in this particular case.